src/libaudcore/vfs_local.cc

```diff
@@ -411,3 +411,119 @@ Index<String> LocalTransport::read_folder(const char * uri, String & error)
 
     return entries;
 }
+
+
+
+#ifdef _WIN32
+#include <sys/stat.h>
+#include <sys/types.h>
+
+bool LocalTransport::get_file_timestamps(const char * filename, int64_t * mtime, int64_t * ctime)
+{
+    StringBuf path = uri_to_filename(filename);
+    if (!path)
+        return false;
+
+    struct _stat64 st;
+    if (_stat64(path, &st) != 0)
+        return false;
+
+    if (mtime)
+        *mtime = st.st_mtime;
+    if (ctime)
+        *ctime = st.st_ctime;
```

Comment:
This isn't creation time. For systems where we don't know how to get creation time, we should just return e.g. -1.